types/dom-mediacapture-record/index.d.ts

```diff
@@ -39,7 +39,7 @@ interface MediaRecorderOptions {
     audioBitsPerSecond?: number | undefined;
     videoBitsPerSecond?: number | undefined;
     bitsPerSecond?: number | undefined;
-    audioBitrateMode?: BitrateMode | undefined;
+    audioBitrateMode?: VideoEncoderBitrateMode | undefined;
```

Comment:
For backwards compat for existing users (since this package is popular), maybe uncomment `BitrateMode` above and make it be `= VideoEncoderBitrateMode`?